src/main/resources/assets/ping/lang/pl_pl.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_comment": "Categories",
+  "ping:key.categories.ping": "Ping",
+
+  "_comment": "Pings",
+  "key.ping": "Otwórz menu pingowania",
+  "ping.key.alert": "Ping (uwaga!)",
+  "ping.key.mine": "Ping (wykop)",
+  "ping.key.look": "Ping (zobacz)",
+  "ping.key.goto": "Ping (idź do)",
+
+  "_comment": "Subtitles",
+  "ping.subtitle.bloop": "Dźwięk pingowania",
+
+  "_comment": "Gui",
+  "ping.pingSelect.title" : "Wybierz typ pingu"
```

Comment:
Where is or will be this string displayed? I think it was supposed to be placed above these 4 types of pings, but it doesn't appear there.

Reply:
It's not displayed anywhere. It's for text-to-speech when opening the Ping GUI.

Reply:
Thank you for the answer. I already force pushed an improved commit.